kedro-datasets/kedro_datasets/api/api_dataset.py

```diff
@@ -15,7 +15,7 @@ class APIDataSet(AbstractDataSet):
     Example:
     ::
 
-        >>> from kedro_datasets.datasets.api import APIDataSet
+        >>> from kedro_datasets.api import APIDataSet
```

Comment:
I was expecting this to be `kedro.datasets` that is mentioned in https://github.com/kedro-org/kedro/issues/1693.

So the `pypi` package name is still `kedro-datasets`, but the folder structure would be `kedro/datasets/xxx` etc.

Reply:
Good catch! I'll go through and replace the examples; should be good to make a note to update the kedro docs as well when the change is made!